packages/react/src/components/UIShell/HeaderMenuButton.js

```diff
@@ -29,6 +30,7 @@ const HeaderMenuButton = ({
     [`${prefix}--header__menu-trigger`]: true,
     [`${prefix}--header__action--active`]: isActive,
     [`${prefix}--header__menu-toggle`]: true,
+    [`${prefix}--header__menu-toggle__hidden`]: true && !isCollapsible,
```

Comment:
Probably you meant just `!isCollapsible`?

Reply:
Yes thanks